src/locales/cs.ts

```diff
@@ -0,0 +1,798 @@
+  //   peter_chanel_priest_patron_of_oceania: 'Saint Peter Chanel, Proto-martyr of Oceania and Patron of Oceania',
+  //   john_brenner_priest: 'Blessed John Brenner, Priest and Martyr',
+  //   paul_of_thebes_hermit: 'Blessed Paul of Thebes, Hermit',
+  //   eusebius_of_esztergom_priest: 'Blessed Eusebius of Esztergom, Priest',
+  //   pavel_peter_gojdic_bishop: 'Blessed Pavel Peter Gojdič, Bishop and Martyr',
+  //   szilard_bogdanffy_bishop: 'Blessed Szilárd Bogdánffy, Bishop and Martyr',
+  //   ladislaus_batthyany_strattmann: 'Blessed Ladislaus Batthyány-Strattmann',
+  //   zoltan_lajos_meszlenyi_bishop: 'Blessed Zoltán Lajos Meszlényi, Bishop and Martyr',
+  //   ceferino_gimenez_malla_martyr: 'Blessed Ceferino Giménez Malla, Martyr',
+  //   john_scheffler_bishop: 'Blessed John Scheffler, Bishop and Martyr',
+    our_lady_of_sorrows_patroness_of_slovakia: 'Panny Marie Bolestné, patronky Slovenska',
+  //   george_of_lydda_martyr_patron_of_england: 'Saint George, Martyr and Patron of England',
+    benedict_of_nursia_abbot_patron_of_europe: 'Sv. Benedikta, opata, patrona Evropy',
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe: 'Sv. Cyrila, mnicha, a Metoděje, biskupa, spolupatronů Evropy',
+  //   cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_slavic_missionaries: 'Saints Cyril and Methodius, Slavic Missionaries',
+    teresa_benedicta_of_the_cross_stein_virgin_copatroness_of_europe: 'Sv. Terezie Benedikty od Kříže, panny, mučednice a spolupatronky Evropy',
+    andrew_apostle_patron_of_scotland: 'Sv. Ondřeje, apoštola a patrona Skotska',
+  //   david_of_wales_bishop_patron_of_wales: 'Saint David, Bishop and Patron of Wales',
+    bridget_of_sweden_religious_copatroness_of_europe: 'Sv. Brigity, řeholnice a spolupatronky Evropy',
+    catherine_of_siena_virgin_copatroness_of_europe: 'Sv. Kateřiny Sienské, panny, učitelky církve a spolupatronky Evropy',
+    catherine_of_siena_virgin_copatroness_of_italy_and_europe: 'Sv. Kateřiny Sienské, panny, učitelky církve a spolupatronky Itálie a Evropy',
+  //   patrick_of_ireland_bishop_patron_of_ireland: 'Saint Patrick, Bishop and Patron of Ireland',
+  //   columba_of_iona_abbot_copatron_of_ireland: 'Saint Columba, Abbot, Missionary and Copatron of Ireland',
+    our_lady_of_guadalupe_patroness_of_the_americas: 'Panny Marie Guadalupské, patronky obou Amerik',
+  //   francis_of_assisi_patron_of_italy: 'Saint Francis of Assisi, Patron of Italy',
+    adalbert_of_prague_bishop_patron_of_poland: 'Sv. Vojtěcha, biskupa, mučedníka a patrona Polska',
+  //   stanislaus_of_szczepanow_bishop_patron_of_poland: 'Saint Stanislaus, Bishop, Martyr and Patron of Poland',
+  wenceslaus_i_of_bohemia_martyr_patron_of_czech_republic: 'Sv. Václava, mučedníka, hlavního patrona českého národa',  // TODO: Add this to all other locales + czechRepublic calendar
```

Comment:
`wenceslaus_i_of_bohemia_martyr_patron_of_czech_republic` - the identifier is inaccurate. St. Wenceslas is not really patron of the *Czech Republic* (the Czech state is non-confessional and has never elected a patron saint), he is a patron of the Czech *nation*. And the Czech word "národ", probably more than the English "nation", refers to an entity primarily ethnical rather than political.
(Which is also the reason why I am not excited that we have an official patron of the Czech nation, as mentioned in igneus/calendarium-romanum#72 , and prefer the older patronal titles referring to a region/territory rather than an ethnical group.)

Reply:
I understand the Czech _words_ and their (primary) meaning, however, those patronage titles you have are quite complicated from my point of view. :wink:

In romcal, we currently support only _nation-wide_/_country-wide_ calendars, no diocesian one. Those would be quite nice addition to romcal, but they would require to be implemented (see #48).

Therefore, from all those patronage titles you have in the Czech proper, which are _for the whole country_ (i.e. Czech Republic)? Are there none? From igneus/calendarium-romanum#72 I understood that St Wenceslaus a patron of all Czech Republic (as in political country), but now you say this isn’t quite true.

Is he just a patron of the the Bohemia (_Česko_, excluding Moravia and Sliezko), or maybe the Bohemian province?

Reply:
The issue about St. Wenceslas is not that his patronage would not be country-wide, but that it's object is "český _národ_" (Czech nation): not a political entity (Czech Republic), but an ethnical one.

Reply:
Argh, now I see what you meant.

Now that complicates some stuff in romcal (e.g. #201), but you are right.

So, St Wenceslaus is patron fo the Czech nation; does it include the Moravians too? Because from my point of view, Czechs are a different nation from Moravians; am I wrong?

Reply:
As discussed in #72, for this purpose Moravians are definitely considered part of the Czech nation. There may be some people asserting a Moravian nation separate from the Czech one, but this way of thinking (is not mainstream and) was definitely not involved when the calendar of Czech and Moravian dioceses was being conceived.

Reply:
Okay, thanks for the thorough explanation.

I’ll use `patron_of_czech_nation` key name suffix.

Anyway, I am tempted to omit `hlavního` from the key value, because I think that in Czech (like in Slovak), the `(principal) patron` is localised simply as `patron` and `co-patron` as `spolupatron`. However, I see a counter-argument that the Czech liturgical books use `hlavní patron` for St Wenceslaus; is `co-patron` localised simply as `patron`? Could check this in the Czech liturgical books please?

Reply:
Just a side question: I have just noticed in the [JPII motu proprio (Latin)](http://www.vatican.va/content/john-paul-ii/la/motu_proprio/documents/hf_jp-ii_motu-proprio_01101999_co-patronesses-europe.html), they used the term _compatronus_, while I thought that _copatronus_ is correct. In the [Latin locale](https://github.com/romcal/romcal/blob/_F-TypeScript/src/locales/la.ts#L473), I used the term _patronus secundarius_. Which is correct according to you

Reply:
> Just a side question: I have just noticed in the JPII motu proprio (Latin), they used the term compatronus, while I thought that copatronus is correct. In the Latin locale, I used the term patronus secundarius. Which is correct according to you

No idea, sorry. It's quite possible that both forms are correct.

> Anyway, I am tempted to omit hlavního from the key value, because I think that in Czech (like in Slovak), the (principal) patron is localised simply as patron and co-patron as spolupatron. However, I see a counter-argument that the Czech liturgical books use hlavní patron for St Wenceslaus; is co-patron localised simply as patron? Could check this in the Czech liturgical books please?

The calendarium-romanum data file you worked with literally reproduces the calendar as printed at the beginning of the Czech breviary and the patronage titles are the same in all currently used liturgical books.

To my knowledge, calendar and liturgical books of Czech and Moravian dioceses don't know an explicit distinction between "patron" and "copatron/secondary patron". The proper calendar calls several saints principal patrons (`hlavní patron` in Czech, `patronus principalis` in [Latin](https://kpmk.eu/kpmkeu/index.php/liturgie/liturgicke-texty/147-texty-narodniho-propria-latinsky)) of various subjects (nation, the three historical countries, dioceses) and there is no second class of patrons in use.

Titles of (all) the feasts of patrons of Europe (which are not proper to the Czech calendar, but shared by all European particular calendars) are, on the other hand, translated simply `patron Evropy`. My understanding has always been that "an equal level of patronage" is shared by all of them and therefore there are only "patroni Evropy" and no "hlavní patron Evropy", but as discussed earlier today, I have yet to clear this question for myself. Currently I cannot exclude the possibility that St. Benedict is actually principal patron of Europe (as you stated earlier today) and official wording of the Czech liturgical calendar for some reason just neglects this distinction.

Either of `patron/spolupatron` or `hlavní patron/patron` is viable, I would probably prefer the second option, as it is a bit closer to the wording used in official liturgical books and clearly distinguishes the principal patron.

Reply:
Thanks for the answer.

> Either of `patron/spolupatron` or `hlavní patron/patron` is viable, I would probably prefer the second option, as it is a bit closer to the wording used in official liturgical books and clearly distinguishes the principal patron.

I see your reasoning. Just a note: in Slovak liturgical books, we use the former, i.e. `patrón` and `spolupatrón`. We clearly distinguish between patrons of Europe that way.

Reply:
(Off topic: they/you've had [printed post-conciliar Latin proper since 1977](https://christ-net.sk/node/3671)! Much envy here.)